Topics/types-of-problem.md

```diff
@@ -0,0 +1,148 @@
+# Frame the problem
+
+# What you will learn 
+In this guide we try to figure out how to frame the kind of problem we want to solve and design it. 
+We will do this by learning to ask the right questions. 
+
+This guide will provide you with many "key questions" of example, which makes sense to ask yourself every time you frame a new data problem. 
+
+These lists of questions are **not exhaustive**, and we welcome Pull Requests that add significantly valid ones.
+
+Every time you start a new project Virgilio recommends that you write a document with the questions and answers explored in this guide, so you always have *clear objectives and characteristics* of the project, as well as being able to *share it easily* with others.
+
+**Do not underestimate this phase at all**, it is perhaps the most important of any project that wants to achieve success!
+
+## Prerequisites
+The previous Purgatorio's guides.
+
+## Time to complete
+20 minutes.
+
+# Index
+- [Recap of ML systems](#Recap-of-ML-systems)
+- [Understand the data](#Understand-the-data)
+- [Set objective and scope](#Set-objective-and-scope)
+- [Usage and Integration](#Integration)
+
+Let's dive right in!
+
+### Recap of ML systems
+
+As we saw in the guide [Introduction to ML systems](Topics/MLSystems.md), there are various ways to classify a problem related to data and learning from them, and in particular the most important classification to frame the problem is as follows: is the problem a supervised or unsupervised learning task?
```

Comment:
I would recommend looking at that image I shared in the group - the ML cycle image which Antinello liked, use the classifications there, what you describe here is limited. It's okay to keep limited but then do mention the other classifications that you have omitted for brevity.